src/main/telemetry/crsf.c

```diff
@@ -493,7 +493,7 @@ static void cRleEncodeStream(sbuf_t *source, sbuf_t *dest, uint8_t maxDestLen)
             c |=  CRSF_RLE_CHAR_REPEATED_MASK;
             const uint8_t fullBatches = (runLength / CRSF_RLE_MAX_RUN_LENGTH);
             const uint8_t remainder = (runLength % CRSF_RLE_MAX_RUN_LENGTH);
-            const uint8_t totalBatches = fullBatches + (remainder) ? 1 : 0;
+            const uint8_t totalBatches = fullBatches + (remainder ? 1 : 0);
```

Comment:
this chances the current logic, right?

Reply:
Yes, before it was like:
```c
const uint8_t totalBatches = (fullBatches + remainder) ? 1 : 0;
```
With this change it's rounding up, which seems correct.

Reply:
It seems to produce different results, for example using `runLength = 1024` produces

```
old code -> fullBatches: 4, remainder: 0 totalBatches: 1
new code -> fullBatches: 4, remainder: 0 totalBatches: 4
```

Reply:
Yeah I don't know how it could have worked before?

Reply:
No idea. We need to test it before merge. Anyone able/willing to do it?

Reply:
Sure looks like a bug. Tested and can confirm it works. 👍

Reply:
Thank you for testing.